WordPress-Aztec-iOS.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'WordPress-Aztec-iOS'
-  s.version          = '1.7.0'
+  s.version          = '1.7.1-beta.1'
```

Comment:
I believe we can release directly the next version 1.7.1, since today we have to release it anyway.